tests/Feature/Api/EncryptedDataTest.php

```diff
@@ -74,46 +69,17 @@ public function test_encrypt_text_field_name_empty()
         $this->assertIsArray($responseData['errors']);
     }
 
-    public function test_encrypt_text_request_id_not_found()
```

Comment:
Instead of this removed test the request_id, add a test to validate uuid is valid and is required in backend:

Reply:
done